src/utils/duplicates.js

```diff
@@ -0,0 +1,23 @@
+import {DUPLICATES_TABLE_HEADERS} from "../constants/duplicatesHeaders";
+
+export const areDuplicatesMerged = (duplicates) => duplicates.data.length < 2;
+
+export const getDuplicatesIds = (duplicates) => duplicates.map((duplicate) => duplicate.studentId);
+
+export const buildMergedItem = (duplicatesSet, selected) => {
+    const item = {};
+    selected.forEach((row, column) => {
+        const field = DUPLICATES_TABLE_HEADERS[column].value;
+        item[field] = duplicatesSet[row][field];
+    });
+    return item;
+};
+
+export const areAllFieldsSelected = (selected) => {
+    for (let i = 0; i < DUPLICATES_TABLE_HEADERS.length; i++) {
+        if (selected[i] === undefined) {
+            return false;
```

Comment:
if you compare boolean just go for (selected[i]), if you compare numbers do (if something > 0), with string do (if something !== ''). It should help: https://github.com/airbnb/javascript